gui/wxpython/rlisetup/frame.py

```diff
@@ -218,7 +218,12 @@ def OnHelp(self, event):
 
     def OnRemove(self, event):
         """Remove configuration file from path and update the list"""
-        confile = self.listfiles[self.listfileBox.GetSelections()[0]]
+        try:
+            confile = self.listfiles[self.listfileBox.GetSelections()[0]]
+        except:
```

Comment:
Please include IndexError in the except, otherwise looks good. Thank you

Reply:
Alright I updated it for all occurence with this exception.